source/Ship.cpp

```diff
@@ -2528,6 +2535,14 @@ bool Ship::CanRefuel(const Ship &other) const
 
 
 
+bool Ship::CanGiveEnergy(const Ship &other) const
+{
+	double toGive = max(200., other.attributes.Get("energy capacity") * 0.2);
+	return energy >= 2 * toGive;
```

Comment:
I wonder if this should use some percentage of the providing ship's capacity instead of the receiving ship's capacity. That way a heavy warship could potentially completely top off the energy of an interceptor, instead of only giving 20% of the interceptor's max, which could be next to nothing for the warship.

Reply:
That would make sense.

Reply:
How about 10% of the sender or 20% of the receiver, whichever is greater?